pandas/io/sql.py

```diff
@@ -482,6 +482,62 @@ def read_sql(
     --------
     read_sql_table : Read SQL database table into a DataFrame.
     read_sql_query : Read SQL query into a DataFrame.
+
+    Examples
+    --------
+    Read data from SQL via either a SQL query or a SQL tablename (latter not
+    possible for SQLite tables)
```

Comment:
I would assume `parse_dates` should still work for sqlite. 

Our docs state

```
Dict of {column_name: arg dict}, where the arg dict corresponds to the keyword arguments of pandas.to_datetime() Especially useful with databases without native Datetime support, such as SQLite.
```

Reply:
Sorry, I will clarify this. What I mean is that SQLite only accepts queries not just table names:
```python
pd.read_sql('SELECT * FROM test_data', conn) # works for SQLite
pd.read_sql('test_data', conn) # does not work for SQLite
```

Reply:
Made explanation more explicit